src/main/java/com/codurance/retropolis/repositories/PostgresCardRepository.java

```diff
@@ -2,20 +2,22 @@
 
 import com.codurance.retropolis.models.Card;
 import com.codurance.retropolis.repositories.mappers.CardMapper;
-import java.sql.PreparedStatement;
-import java.util.Objects;
-import javax.sql.DataSource;
 import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.jdbc.support.GeneratedKeyHolder;
 import org.springframework.jdbc.support.KeyHolder;
 import org.springframework.stereotype.Repository;
 
+import javax.sql.DataSource;
+import java.sql.PreparedStatement;
+import java.util.Objects;
+
 @Repository
 public class PostgresCardRepository implements CardRepository {
 
   private final String INSERT_CARD = "insert into cards (text, username, column_id) values (?,?,?)";
   private final String SELECT_CARD = "select * from cards where id = ?";
   private final String DELETE_CARD = "delete from cards where id = ?";
+  private final String UPDATE_CARD = "update cards set text = ? where id = ?";
```

Comment:
Good. This is clearly labelled SQL so that it is clear what each SQL code is doing.